tools/test-tools/package.json

```diff
@@ -1,6 +1,7 @@
 {
-	"name": "@fluidframework/test-tools",
-	"version": "3.0.0",
+	"name": "@fluid-private/test-tools",
```

Comment:
if we are renaming would be a good time to give a more descriptive name if we can come up with one. this is really about infrastructure, and really only does assignment of tinyclicious ports.

now that i think about it, it also seems similar to how we do assert tagging, could this maybe just move to policy check?

Reply:
Not sure it's that similar, the code in this package gets called by the npm script to create the port mapping file, but also by jest config files to get the appropriate port when running jest tests. Policy-check in general is something we can run in parallel and doesn't block building or running tests, right?

I agree there's probably a better place for this functionality than its own package though. Maybe test-utils (although doesn't quite match the package intention per its README), or a separate `/test-utils` export in client-utils? A new `jest-test-setup` analogous to the mocha one seems overkill.